pkg/agent/resourcemanager/outofband/types.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2022 The Katalyst Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package outofband
+
+const (
+	MetricAddPodTimeout    = "ORM_add_pod_timeout"
+	MetricDeletePodTImeout = "ORM_delete_pod_timeout"
+
+	MainContainerNameAnnotationKey = "kubernetes.io/main-container-name"
```

Comment:
Do we need to support the main container annotation for ORM?

Reply:
> Do we need to support the main container annotation for ORM?

Not necessary, first container in container list will be defaulted as the main container, but we can keep it as a feature.  What do you think?

Reply:
In vanilla K8s, all containers are treated as main containers. If we keep this manner, will there be any problems in the colocation scenario?

Reply:
To remain consistent with QRM, we can retain this usage. In the long term, when the k8s version is higher, we can use the native sidecar container feature provided by k8s. https://kubernetes.io/blog/2023/08/25/native-sidecar-containers/